docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,184 @@
+# Deploy Nebula Graph with Docker Compose
+
+There are multiple ways to deploy Nebula Graph, but using Docker Compose is usually considered to be a fast starter.
+
+## Reading guide
+
+If you are reading this topic with the questions listed below, click them to jump to their answers.
+
+* [What do I need to do before deploying Nebula Graph?](#prerequisites)
+* [How to fast deploy Nebula Graph with Docker Compose?](#how-to-deploy)
+* [How to check the status and ports of the Nebula Graph service?](#check-the-nebula-graph-service-status-and-ports)
+* [How to check the data and logs of the Nebula Graph service?](#check-the-service-data-and-logs)
+* [How to stop the Nebula Graph Services?](#stop-the-nebula-graph-services)
+* [What are the other ways to install Nebula Graph?](#other-ways-to-install-nebula-graph)
+
+## Prerequisites
+
+* You have installed the following applications on your host.
+  
+  | Application    | Recommended version | Official installation reference                                    |
+  | -------------- | ------------------- | ------------------------------------------------------------------ |
+  | Docker         | Latest              | [Install Docker Engine](https://docs.docker.com/engine/install/)   |
+  | Docker Compose | Latest              | [Install Docker Compose](https://docs.docker.com/compose/install/) |
+  | Git            | Latest              | [Download Git for Linux](https://git-scm.com/download/linux)       |
```

Comment:
why Git for Linux only?

Reply:
Nebula Graph supports only Linux for now.